sdk/attestation/azure-security-attestation/azure/security/attestation/_models.py

```diff
@@ -313,491 +328,425 @@ def policy_claims(self):
 
     @property
     def verifier_type(self):
-        #type:() -> Union[str, None]
-        """ Returns the verifier which generated this attestation token.
+        # type: () -> str
+        """Returns the verifier which generated this attestation token.
 
-        :rtype: str or None
+        :rtype: str
         """
         return self._verifier_type
 
     @property
     def policy_signer(self):
-        #type:() -> Union[AttestationSigner, None]
-        """ Returns the signing certificate which was used to sign the policy
+        # type: () -> Union[AttestationSigner, None]
+        """Returns the signing certificate which was used to sign the policy
         which was applied when the token was generated.
 
-        :rtype: azure.security.attestation.AttestationSigner or None
+        :rtype: ~azure.security.attestation.AttestationSigner or None
         """
-        if self._policy_signer:
-            return AttestationSigner._from_generated(self._policy_signer)
-        return None
+        return AttestationSigner._from_generated(  # pylint: disable=protected-access
+            self._policy_signer
+        )
 
     @property
     def policy_hash(self):
-        #type:() -> Union[str, None]
-        """ Returns the base64url encoded SHA256 hash of the Base64Url encoded
+        # type: () -> str
+        """Returns the base64url encoded SHA256 hash of the Base64Url encoded
```

Comment:
I noticed that another instance of "Base64Url" was lower-cased, but not this one
```suggestion
        """Returns the base64url encoded SHA256 hash of the base64url encoded
```